extensions/sql-database-projects/src/typings/vscode-mssql.d.ts

```diff
@@ -5,613 +5,620 @@
-        ownerUri: string;
-        taskExecutionMode: TaskExecutionMode;
-    }
-
-    export interface GenerateDeployScriptParams {
-        packageFilePath: string;
-        databaseName: string;
-        sqlCommandVariableValues?: Record<string, string>;
-        deploymentOptions?: DeploymentOptions;
-        ownerUri: string;
-        taskExecutionMode: TaskExecutionMode;
-    }
-
-    export interface GenerateDeployPlanParams {
-        packageFilePath: string;
-        databaseName: string;
-        ownerUri: string;
-        taskExecutionMode: TaskExecutionMode;
-    }
-
-    export interface GetOptionsFromProfileParams {
-        profilePath: string;
-    }
-
-    export interface ValidateStreamingJobParams {
-        packageFilePath: string;
-        createStreamingJobTsql: string;
-    }
-
-    export interface SchemaCompareGetOptionsParams { }
-
-    export interface SchemaCompareOptionsResult extends ResultStatus {
-        defaultDeploymentOptions: DeploymentOptions;
-    }
-
-    export interface ITreeNodeInfo extends vscode.TreeItem {
-        readonly connectionInfo: IConnectionInfo;
-        nodeType: string;
-        metadata: ObjectMetadata;
-        parentNode: ITreeNodeInfo;
-    }
-
-    export const enum MetadataType {
-        Table = 0,
-        View = 1,
-        SProc = 2,
-        Function = 3
-    }
-
-    export interface ObjectMetadata {
-        metadataType: MetadataType;
-
-        metadataTypeName: string;
-
-        urn: string;
-
-        name: string;
-
-        schema: string;
-    }
-
-    /**
-     * Azure functions binding type
-     */
-    export const enum BindingType {
-        input,
-        output
-    }
-
-    /**
-     * Parameters for adding a SQL binding to an Azure function
-     */
-     export interface AddSqlBindingParams {
-        /**
-         * Aboslute file path of file to add SQL binding
-         */
-        filePath: string;
-
-        /**
-         * Name of function to add SQL binding
-         */
-        functionName: string;
-
-        /**
-         * Name of object to use in SQL binding
-         */
-        objectName: string;
-
-        /**
-         * Type of Azure function binding
-         */
-        bindingType: BindingType;
-
-        /**
-         * Name of SQL connection string setting specified in local.settings.json
-         */
-        connectionStringSetting: string;
-    }
-
-    /**
-     * Parameters for getting the names of the Azure functions in a file
-     */
-    export interface GetAzureFunctionsParams {
-        /**
-         * Absolute file path of file to get Azure functions
-         */
-        filePath: string;
-    }
-
-    /**
-     * Result from a get Azure functions request
-     */
-    export interface GetAzureFunctionsResult extends ResultStatus {
-        /**
-         * Array of names of Azure functions in the file
-         */
-        azureFunctions: string[];
-    }
+	import * as vscode from 'vscode';
```

Comment:
Please undo the whitespace changes here - this should be directly copied from the MSSQL extension with the changes made (don't format it here). 

We should fix it so that the formatting is the same in both repos, but for now let's at least stay consistent with what the original source is.